tutorials/preprocessing/25_background_filtering.py

```diff
@@ -316,15 +317,15 @@
 # is constant) but small in the pass-band. Unlike zero-phase filters, which
 # require time-shifting backward the output of a linear-phase filtering stage
 # (and thus becoming non-causal), minimum-phase filters do not require any
-# compensation to achieve small delays in the pass-band. Note that as an
-# artifact of the minimum phase filter construction step, the filter does
-# not end up being as steep as the linear/zero-phase version.
+# compensation to achieve small delays in the pass-band.
 #
 # We can construct a minimum-phase filter from our existing linear-phase
 # filter with the :func:`scipy.signal.minimum_phase` function, and note
-# that the falloff is not as steep:
+# that the falloff is not as steep (though here we use a temporary workaround
+# version vendored by MNE until https://github.com/scipy/scipy/pull/19706 is
+# merged and released):
 
-h_min = signal.minimum_phase(h)
+h_min = minimum_phase(h, half=False)
```

Comment:
not sure we want to talk about "vendoring" in our tutorials.  Seems like a detail users shouldn't need to know about. Maybe instead just mention which SciPy / MNE versions & params are necessary to get the correct behavior?